### DISCUSSION TITLE
not ripping multiple images in a single post.

### DISCUSSION THREAD

**Discussion by superfan12345** [UltimaHoarder/UltimaScraper#1858] [General]
whenever I use this it seems to not get multiple images. it only gets the first. from a single post. 

so for example a creator uploads 10 images in a single post. it will only get the 1st image and not the other 9

**Comment by superfan12345** [comment]
anyone?

**Comment by superfan12345** [comment]
once again. figured it out.

**Reply by cipher2022** [reply]
What did you do to figure it out?
